src/main/java/org/littleshoot/proxy/extras/SelfSignedSslEngineSource.java

```diff
@@ -150,8 +234,7 @@ public void checkServerTrusted(X509Certificate[] arg0,
             sslContext = SSLContext.getInstance(PROTOCOL);
             sslContext.init(keyManagers, trustManagers, null);
         } catch (final Exception e) {
-            throw new Error(
-                    "Failed to initialize the server-side SSLContext", e);
+            throw new Error("Failed to initialize the server-side SSLContext", e);
```

Comment:
As above, I'm thinking we probably don't want to throw a generic Exception type here (and definitely not Error). I'd rather catch specific checked exceptions and throw appropriate exceptions up the stack.